tests/foreman/rhai/test_rhai.py

```diff
@@ -98,3 +99,25 @@ def test_rhai_client_setup(self):
             Navigator(self.browser).go_to_insights_systems()
             result = self.rhai.view_registered_systems()
             self.assertIn("1", result, 'Registered clients are not listed')
+
+    def test_org_selection_for_rhai(self):
+        """@Test:- Verify that user attempting to access RHAI is directed to
+        select an Organization if there is no organization selected
+
+        @Feature: In order to use Access Insights, user must select an
+        organization
+
+        @Assert: 'Organization Selection Required' message must be displayed if
+        the user tries to view Access Insights overview without selecting an
+        org
+        """
+
```

Comment:
Nitpick: move the triple quotes down a line, like so:

    @Assert: 'Organization Selection Required' message must be displayed if
    the user tries to view Access Insights overview without selecting an
    org

    """
    with Session(self.browser) as session: